app/graphql/types/stage_type.rb

```diff
@@ -0,0 +1,20 @@
+module Types
```

Comment:
There is probably a slicker way to implement this, there is an opportunity for some refactoring, but this works. I considered a scalar type, but that didn't seem appropriate since this is composed of scalar types.